### ISSUE TITLE
Internal server error when saving a story

### ISSUE THREAD

**Issue by kojaa** [swissspidy/web-stories-wp#13720]
## Bug Description

When trying to save a story, the server 500 error occurs if the publisher logo on the story remains unchanged. If the logo gets updated, the issue does not happen. This bug also didn't happen in previous versions of the plugin.

## Expected Behaviour

The story should be saved without having to change the publisher logo every time.

## Steps to Reproduce

Create a story, insert slide, insert publisher logo and save. Add another slide and click on the update button.

## Screenshots

Video of the issue: https://www.youtube.com/watch?v=kksd3mude8k

## Additional Context

- Plugin Version: 1.36.0
- WordPress Version: 6.5.3
- Operating System: Windows 11
- Browser:  Firefox Developer 127.0b9 (64-bit) | Google Chrome Version 125.0.6422.113 (Official Build) (64-bit)
- Website hosted on: CloudLinux 8
- PHP Version: 8.2.19

**Comment by swissspidy** [comment]
Heya, thanks for your report! I appreciate that you took the extra step to record a video as well.

This is an unusual issue, and not something our tests or any other user have reported so far. So it could be isolated to your specific environment.

If you go to the "Request" tab, what's the value that it tries to send for the `web_stories_publisher_logo`?

Also, are you using an external object cache such as Memcached or Redis on your server?

**Comment by swissspidy** [comment]
The thing is, WordPress itself does short-circuit if the sent value is identical to what's already stored in the database. This happens here:

https://github.com/WordPress/wordpress-develop/blob/8c4be2687cc4c44e21aaf99feb167c93db31aa85/src/wp-includes/rest-api/fields/class-wp-rest-meta-fields.php#L384-L388

The error message indicates that this check fails on your system, so I'd be curious to learn more about your setup, such as the type of database you use, and what exactly is stored on your database. I assume the value is `1073` everywhere.

Also, I see you're using some other potentially related plugins such as ACF. Does this issue still happen when you temporarily disable all other plugins?

**Comment by kojaa** [comment]
Hi and thank you for your fast response!

You have mentioned REST in your previous comment and the potential same value to be everywhere in database, which led us to check the database entries carefully. And we have found that we had same `meta_key` entries for `web_stories_publisher_logo` and `thumbnail_id`. 

<img width="665" alt="Screenshot_2024-06-04_at_11 17 01" src="https://github.com/GoogleForCreators/web-stories-wp/assets/33633522/f9b80053-9880-46d1-bad1-0cd184b08ca3">

At one point, we are not sure where and how the meta key got duplicate entry which was causing the error, we assume that your plugin only expects this to be one entry. That is why we had to re-select logo every time we wanted to update the story.

Thanks again!
